tests/testing-framework/Cargo.toml

```diff
@@ -13,7 +13,7 @@ nix = "0.26.1"
 regex = "1.10.2"
 reqwest = { workspace = true }
 temp-dir = "0.1.11"
-test-environment = { git = "https://github.com/fermyon/conformance-tests", branch = "main" }
+test-environment = { git = "https://github.com/fermyon/conformance-tests", rev = "2bb3eb83467a9e9595fa0fd11d7a49fa0300b113" }
```

Comment:
Should conformance-tests move into this repo?

Reply:
The point of moving them out of this repo was to facilitate reuse across multiple runtimes, so I don't think so.

Reply:
We reuse other spin repo code in other runtimes... :shrug:

I'm just always a little wary of having test code separated from the (main) thing its testing.

Reply:
As the main person working on this, I can see it has very much helped me keeping these separate. If others find it difficult to contribute because of this separation, we certainly should reconsider my decision, but until there's a concrete reason to move this closer, I'd strongly prefer to continue to keep them apart.